src/connector/client.js

```diff
@@ -319,11 +328,19 @@ Zotero.GoogleDocs.Client.prototype = {
 				return field;
 			}
 		}
+		if (selectedFieldID.startsWith("broken=")) {
+			this.isInOrphanedField = true;
+			if (showOrphanedCitationAlert && !this.orphanedCitationAlertShown) {
+				await Zotero.GoogleDocs.UI.displayOrphanedCitationAlert();
```

Comment:
Can we detect when the user cancels out of the citation dialog on a broken citation and leave the broken citation in place in that case? The current behavior makes it easy to wipe out the citation before you've had a chance to replace it. I did that multiple times just while testing this, and I think people will be likely to do that when encountering this situation before they take time to read the dialog.